spec/grub_cfg_spec.rb

```diff
@@ -25,4 +25,29 @@
       )
     end
   end
+
+  describe "#boot_entries" do
+    it "gets boot entries list with title: and path: elements" do
+      expect(config.boot_entries).to eq(
+        [
+          {
+            title: "openSUSE Leap 42.1",
+            path: "openSUSE Leap 42.1"
+          },
+          {
+            title: "openSUSE Leap 42.1, with Linux 4.1.12-1-default",
+            path: "Advanced options for openSUSE Leap 42.1>openSUSE Leap 42.1, with Linux 4.1.12-1-default"
+          },
+          {
+            title: "openSUSE Leap 42.1, with Linux 4.1.12-1-default (recovery mode)",
+            path: "Advanced options for openSUSE Leap 42.1>openSUSE Leap 42.1, with Linux 4.1.12-1-default (recovery mode)"
+          },
+          {
+            title: "halt",
+            path: "halt"
+          }
+        ]
+      )
+    end
+  end
```

Comment:
If the parser is "Private only, internal" and does not inherit from anything then the method should not be there at all. Or it should inherit from CFA::Parser (which does not exist) or from CFA::AugeasParser where [serialize](http://www.rubydoc.info/github/config-files-api/config_files_api/CFA/AugeasParser#serialize-instance_method) is an instance method.

I know that Ruby really knows only one kind of methods and does not care who is the receiver of a message, but here we are concerned with communicating the API to a human.